modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc

```diff
@@ -34,23 +34,24 @@ If the cluster is configured with an intermediate certificate, then the whole ch
 
 == Adding new CA certificates into {prod-short}
 
-The following procedure is applicable for already installed and running instances and for instances that are to be installed.
+Certificate files are typically stored as Base64 files, with extensions such as `.pem`, `.crt`, `.ca-bundle`and others. But they can also be binary-encoded, for example, as `.cer` files. All Secrets that hold certificate files should use the Base64-encoded certificate rather than binary-encoded. The following procedure is applicable for already installed and running instances and for instances that are to be installed.
```

Comment:
```suggestion
Certificate files are typically stored as Base64 files, with extensions such as `.pem`, `.crt`, `.ca-bundle` and others. But they can also be binary-encoded, for example, as `.cer` files. All Secrets that hold certificate files should use the Base64-encoded certificate rather than binary-encoded. The following procedure is applicable for already installed and running instances and for instances that are to be installed.
```